CHANGELOG.md

```diff
@@ -10,11 +10,13 @@ _None_
 
 ### New Features
 
-_None_
+- `promo_screenshots` now checks that the fonts—referenced via `font-family` in all the stylesheets referenced in the config file—are installed before starting, and prompt to install them if they are not. This check is enabled by default now but can be disabled/skipped if it causes any issue. [#429]
+- `promo_screenshots` now supports config files to be written in `YAML` in addition to still supporting `JSON`. [#429]  
+  Note that while this is technically not a breaking change—because YAML is a superset of JSON—the old way of parsing the JSON file had a loophole that incorreclty allowed `// comments` in the JSON file (despite this not being allowed in the JSON standard). If your config files relied on this bug and used `// comments`, you will have to replace them with `# comments` instead so that they can be parsed correctly by the YAML parser.
```

Comment:
I hesitated to put that item under Breaking Change.

Technically speaking, the breaking change is only true if we relied on a bug in the ruby JSON parser ignoring `//` comments, while it should never have according to the JSON spec, so it's not a breaking of the official API contract we had so far, and clients which were using `// comment` were relying on this glitch/bug not on something official from our API contract.

In practice though, I know that most of our `screenshots.json` config files did include `// comments` before each entry to document which screenshot that entry was for… so it will still be breaking and require changes in most of our clients adopting this versions and which relied on that bug… So should that mean a `7.0.0` bump after all?

Reply:
Minor (🔍): A small type here: `incorrectly`

Reply:
Nice catch!
```suggestion
  Note that while this is technically not a breaking change—because YAML is a superset of JSON—the old way of parsing the JSON file had a loophole that incorrectly allowed `// comments` in the JSON file (despite this not being allowed in the JSON standard). If your config files relied on this bug and used `// comments`, you will have to replace them with `# comments` instead so that they can be parsed correctly by the YAML parser.
```

Reply:
Actually, instead of all that, I just pushed a442ee8 which still uses `JSON.parse` if the file has a `.json` extension, and use `YAML.load_file` otherwise (i.e. for `.yml` and `.yaml`). This way, we will still be lenient and support `// comments` if the file is in `.json`, making sure this won't be a breaking change _even_ for clients that relied on that glitch and ruby parser leniency.